.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,26 @@
+name: Node CI
+
```

Comment:
That is just a file test with Travis before. I will remove it after. 
and we will clean commit on Github because I had pushed so many commits on Github.